MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -3106,7 +3118,6 @@
 		DAAE5F321EAE4C4700832871 /* Localizable.strings */ = {
 			isa = PBXVariantGroup;
 			children = (
-				DAAE5F311EAE4C4700832871 /* Base */,
```

Comment:
Transifex is currently configured to expect Base localization:

https://github.com/mapbox/mapbox-navigation-ios/blob/1783c9b6e2556b41b79851522dfe943b108af2e2/.tx/config#L9

I was under the impression that Base localization is a best practice. Xcode even issued a warning for us to adopt it at one point. Is it unsupported by SPM?

Reply:
I was under the same expression, but the situation is a bit more complicated: Base internationalization is still a preferred approach, but it is designed only for .storyboard and .xib files. From the docs:
```
Base internationalization separates user-facing strings from .storyboard and .xib files. It relieves localizers of the need to modify .storyboard and .xib files in Interface Builder. Instead, an app has just one set of .storyboard and .xib files where strings are in the development language—the language that you used to create the .storyboard and .xib files. These .storyboard and .xib files are called the base internationalization. When you export localizations, the development language strings are the source that is translated into multiple languages. When you import localizations, Xcode generates language-specific strings files for each .storyboard and .xib file. The strings files don't appear in the project navigator until you import localizations or add languages.

In Xcode 5 and later, base internationalization is enabled by default. If you have an older project, enable base internationalization before continuing, as described in Enabling Base Internationalization.
```
It clearly says that base internationalization is just for .storyboard and .xib files. I tried to verify this information and created a new project. When I try to localize a xib file, Xcode allows me to select base localization and all the other supported languages I've added in settings. If I choose a string file, then Xcode doesn't show Base localization. This matches with the documentation. 

Having said above, I will:
- [ ] Verify Transifex integration and see how we can fix it. 

Thanks for letting me know about Transifex, gotta read more about it.

Reply:
We probably need to update extract_localizable.sh to put non-IB strings in en.lproj instead of Base.lproj:

https://github.com/mapbox/mapbox-navigation-ios/blob/68f671d7633d8201cd01cc7f20dccc38f43ae4e5/scripts/extract_localizable.sh#L9